python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -737,6 +744,7 @@ def __call__(self):
     # it back to 0. This bursty behavior should be smoothed by the delay
     # parameters.
     for _ in range(5):
+        ray.get(signal.send.remote(clear=True))
         assert check_autoscale_num_replicas(controller, "A") == num_replicas
         refs = [handle.remote() for _ in range(100)]
         signal.send.remote()
```

Comment:
Why don't you clear it here?

Reply:
I think setting `clear=True` is "reblocking" the signal actor. Then `signal.send.remote()` unblocks it.

Reply:
It unblocks and then re-blocks it ([source](https://github.com/ray-project/ray/blob/1cf24b40017ab26777c65682afc9ae531fee41d9/doc/source/ray-core/doc_code/actor-sync.py#L12-L15)). If we expect all the users to be queued by the time we call `signal.send.remote()`, we should be able to re-block immediately.